docs/en/ingest-management/elastic-agent/configuration/providers/elastic-agent-providers.asciidoc

```diff
@@ -35,8 +35,10 @@ providers:
       - item: key2
 ----
 
-Explicitly disable a provider by setting `enabled: false`. All providers
-are prefixed without name collisions. The name of the provider is in the key in the configuration.
+Providers are disabled by default.
+Explicitly enable a provider by setting `enabled: true`.
```

Comment:
Even this will not cause a provide to run. The provider must be referenced in the policy or it will not run.